addons/opsportal/1.1.x/opsportal.yaml

```diff
@@ -29,6 +29,8 @@ spec:
     chart: opsportal
     repo: https://mesosphere.github.io/charts/stable
     version: 0.3.12
+    valuesRemap:
```

Comment:
how come in this one you renamed the file w/o the revision number but all the other addons maintain them?

Reply:
That is how the addon exists on master: https://github.com/mesosphere/kubernetes-base-addons/tree/master/addons/opsportal/1.1.x

Reply:
I'll align opsportal with other addons after this release… initial thinking was that this repo does only support snapshots via tags anyway so history is done via git tags, but given its the only addon doing it like that its confusing and should be aligned with other addons.